### DISCUSSION TITLE
"Year (upload year/release year)" clarification.

### DISCUSSION THREAD

**Discussion by DwarfJockey** [RandomNinjaAtk/docker-lidarr-extended#79] [Q&A] [answered]
The README states that nfo's are tagged with "Year (upload year/release year)", but all I can see is the upload year of the video file. How could I replace that with the release year of the videos. It would be cool to sort videos by decade they came out instead of when they were uploaded to Youtube.

**Comment by RandomNinjaAtk** [comment, marked as answer]
It only adds a single year to the NFO file, so not both. What your asking for seems like it should be more of a software feature in an app. Like, show all videos from the 2010s.

**Reply by DwarfJockey** [reply]
Yeah, that's what I'm trying to do. The only year is the Youtube upload year. I want the original release year for the music video. Is that a setting somewhere to prefer that?

By the way, thanks, your music video script is excellent.

**Reply by RandomNinjaAtk** [reply]
There really isn't any music video database info that is super reliable, Video Upload year/date is probably more reliable to show when a video was released...

If your thinking original release year of the song or album itself, and not the video... That would likely be way to complicated to try to make work, because then a video would have to be associated with an actual track/album.

The only sources for years are Musicbrainz,Tidal, or IMVDB sites, where the videos are sourced from... So if you can provide an example of where the NFO year is different from the sites listed year for the same video, then it might be worth while investigating to make a change for that... But the year would only be updated to match the sources site year, not necessarily the release year for a song/album...

Edit: I've also seen cases where the source site (imvdb), did not actually provide a year for a video, so in those cases you have to use the Upload date/year.

**Reply by DwarfJockey** [reply]
Once upon a time I had ambitions to write a script to serve a similar use where I was comparing Musicbrainz data to search for videos on Youtube by artist and song title and then writing an nfo using the earliest version of the artist/song title combo's release date from Musicbrainz. I eventually gave up on it cause it liked to download a lot of lyric and fan made videos, but could a similar solution work here?

**Reply by RandomNinjaAtk** [reply]
PRs welcome
